apolloschurchapp/src/ui/HomeTabHeader.js

```diff
@@ -20,6 +19,13 @@ const GreetingText = styled(({ theme }) => ({
   color: theme.colors.text.primary,
 }))(H3);
 
+const Background = styled({
+  width: '100%',
+  marginTop: -80,
+  aspectRatio: 375 / 357,
```

Comment:
Where do these numbers come from?

Reply:
```suggestion
  aspectRatio: 375 / 357, // the image is 375px x 357px
```

`ImageBackground` component requires a determined `width` and `height` within `styles`. With `aspectRatio`, we can provide either just a `width` or `height` and the other attribute is determined off of the `aspectRatio`. The image is 375x357, so just divided the two numbers here for aspect ratio.

`-80` was to offset the image to best match the Figma design - some of the image bleeds under the header instead of completely below it.